Employee-Microservice/src/main/java/base/service/EmployeeServiceImpl.java

```diff
@@ -83,8 +80,7 @@ public List<Employee> findAll() {
                     .map(DtoConverter::entityToModel)
                     .toList();
         } catch (Exception e) {
-            log.error("something went wrong to fetch Employees with Error:: {}", e.getMessage());
-            throw new RuntimeException(e.getMessage());
+            throw new EmployeeNotFoundException("something went wrong to fetch Employees with Error: "+e.getMessage());
```

Comment:
Did you run all the test cases and made sure this change is not breaking?
If not please go for it once